app/assets/javascripts/application.js.erb

```diff
@@ -373,3 +373,34 @@ $(document).on('click', '[href="#activity_log"]', function () {
 
     return false;
 });
+
+<% if TeSS::Config.feature['sticky_navbar'] %>
+/**
+ * Created by Kenneth Rioja on 02.07.2025
+ *
+ * Sticky Navbar
+ */
+
```

Comment:
I would not toggle the feature this way, since it would involve rebuilding the JS/CSS if the feature is toggled.

Instead you can add a class or data attribute to the header (or `body` element even) in the ERB templates if the feature is enabled, and then check for that class/attribute dynamically with a JS selector.